core/capabilities/ccip/oraclecreator/bootstrap.go

```diff
@@ -37,13 +40,91 @@ import (
 
 var _ cctypes.OracleCreator = &bootstrapOracleCreator{}
 
+// bootstrapOracle wraps a CCIPOracle (the bootstrapper) and manages RMN-specific resources
+type bootstrapOracle struct {
+	baseOracle      cctypes.CCIPOracle
+	peerGroupDialer *peerGroupDialer
+	rmnHomeReader   ccipreaderpkg.RMNHome
+	started         bool
```

Comment:
Is this necessary? I think with the current lifecycle, once `Close` is called the object should be discarded forever.

Reply:
This is also checking if the oracle has already been started. But I think we can remove it this logic as start should not be called more than once anyway.